src/database/index.tsx

```diff
@@ -838,3 +839,76 @@ export const deleteScenario = async (
     );
   }
 };
+
+const scenarioMigrationCollectionId = "scenarios";
+
+export const getMigrationScenarios = async () => {
+  const db = await getDb();
+  return db.collection(scenarioMigrationCollectionId).get();
+};
+
+export const getMigrationFacilities = async (scenarioId) => {
+  const db = await getDb();
+  return db
+    .collection(scenarioMigrationCollectionId)
+    .doc(scenarioId)
+    .collection(facilitiesCollectionId)
+    .get();
+};
+
+export const getMigrationModelVersions = async (scenarioId, facilityId) => {
+  const db = await getDb();
+  return db
+    .collection(scenarioMigrationCollectionId)
+    .doc(scenarioId)
+    .collection(facilitiesCollectionId)
+    .doc(facilityId)
+    .collection(modelVersionCollectionId)
+    .get();
+};
+
+export const addFacilityCapacity = async ({
+  scenarioId,
+  facilityId,
+  modelVersionId,
+  facilityCapacity,
+  modelInputs,
+  write,
+}) => {
+  const newInputs = { ...modelInputs, facilityCapacity };
+  const timestamp = currrentTimestamp();
+  newInputs.updatedAt = timestamp;
+  delete newInputs.facilityOccupancyPct;
+
+  const db = await getDb();
+  let docToUpdate = db
+    .collection(scenarioMigrationCollectionId)
+    .doc(scenarioId)
+    .collection(facilitiesCollectionId)
+    .doc(facilityId);
+  let updateData = { modelInputs: newInputs, updatedAt: timestamp };
+  if (modelVersionId) {
+    docToUpdate = docToUpdate
+      .collection(modelVersionCollectionId)
+      .doc(modelVersionId);
+    updateData = {
+      ...newInputs,
+      facilityOccupancyPct: firebase.firestore.FieldValue.delete(),
```

Comment:
Why do we add this delete when we're talking about a model version?

Reply:
for the facility, we are replacing the entire value of `facility.modelInputs` with this new input object, whereas for a model version there is no `modelInputs` key so it just merges the objects together. To remove this field I need to send an explicit delete command.

Reply:
this is now moot anyway because after talking with @jovergaag I realized it was probably premature to delete this field anyway, since it will break stuff in existing deployment while we're waiting for her feature to be released. that should make this migration non-destructive and we can do another pass later to clean up the unused field after we are sure that everything is copacetic post-release